### ISSUE TITLE
Expired Trial Key Fails Silently without Error

### ISSUE THREAD

**Issue by abhisek** [safedep/vet#24]
## Problem

When the configured trial key is expired, scan runs without error but enrichment fails and hence no result is displayed. Only when we run with verbose logging can we know that the trial key has expired.

## Solution

Introduce key verification and verify key before starting scan. Fail if key is invalid. This should be done only for scan command and not for offline analysis like query command

**Comment by abhisek** [comment]
Duplicate of #7
